.github/workflows/perf.yml

```
@@ -0,0 +1,206 @@
+          GH_TOKEN: ${{ github.token }}
+          PR_NUMBER: ${{ github.event.pull_request.number }}
+        run: |
+          MARKER="<!-- raven-benchmark-comparison -->"
+
+          # Build the comment body based on comparison results.
+          # BODY content must start at column 0 — GFM treats 4+ leading spaces as code blocks.
+          if [ "${{ steps.critcmp.outputs.no_baseline }}" = "true" ]; then
+            BODY="${MARKER}
+## Benchmark Comparison
+
+No \`main\` baseline found — benchmark comparison skipped.
```

actionlint (1.7.10)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)